roles/openshift_storage_glusterfs/tasks/heketi_deploy_part2.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Create heketi DB volume
-  command: "{{ glusterfs_heketi_client }} setup-openshift-heketi-storage --image {{ glusterfs_heketi_image}}:{{ glusterfs_heketi_version }} --listfile /tmp/heketi-storage.json"
+  command: "{{ glusterfs_heketi_client }} setup-openshift-heketi-storage --listfile /tmp/heketi-storage.json"
   register: setup_storage
```

Comment:
This is bug. New version of heketi does not need image anymore

Reply:
It fails with "image flag is unknown". are you sure this is needed?

Reply:
I should probably clarify: yes this should get fixed, just in a separate PR. :)